DokuwikiXMLExport.php

```diff
@@ -100,43 +119,54 @@ private function splitConfigToArray($config)
     public function generateXMLExport($start, $count)
     {
         $exporter = Exporter::create(Exporter::TYPE_XML, $count);
+
         $total = count($this->pages);
+        $submittedCount = $count;
```

Comment:
Probably more clear if you rename the method argument from `$count` to `$submittedCount` and then do `$count = min($total, $submittedCount);`

Reply:
Agreed

Reply:
:white_check_mark: